internals/demo.sh

```diff
@@ -19,15 +32,21 @@ DEMO_PROMPT="$BLUE~/demo-repository$COLOR_RESET$ "
 ### initial setup ###
 
 p "git init"
-git init > /dev/null
+git init &> /dev/null
 echo "Initialized empty Git repository"
 BRANCH_NAME=$(git branch --show-current)
 DEMO_PROMPT="$BLUE~/demo-repository$GREEN($BRANCH_NAME)$COLOR_RESET$ "
 
 # the following files are not needed for the demo
-pe "rm -r .git/hooks"
-pe "rm -r .git/info/exclude"
-pe "rm -r .git/description"
+if [[ -d ".git/hooks" ]]; then
```

Comment:
Thanks. Which version of Git do you use?
After the initial creation of a repo these folders should definitely exists.

Reply:
I'm currently using version `2.38`
![image](https://user-images.githubusercontent.com/23143673/196003560-12fc6ba7-820e-4206-8b7a-3f0a032d8d19.png)

Reply:
Ok interesting apparently my `.gitconfig` setting for `init` was pointing to an empty template directory. Therefore the files and folders were never created on my system.
Should I leave the checks in or remove them because they are usually not necessary?

Reply:
let's leave them. ;)